telemetry/definitions/commonDefinitions.json

```diff
@@ -142,6 +147,63 @@
             "name": "aws_reportPluginIssue",
             "description": "Report an issue with the plugin"
         },
+        {
+            "name": "cloudwatchlogs_openGroup",
+            "description": "Open the CloudWatch Logs group window",
+            "metadata": [{ "type": "result" }]
+        },
+        {
+            "name": "cloudwatchlogs_openStream",
+            "description": "Open a CloudWatch Logs stream in the window",
+            "metadata": [{"type": "result"}]
+        },
+        {
+            "name": "cloudwatchlogs_downloadToFile",
+            "unit": "Bytes",
+            "description": "Download a log stream to a file. Value indicates the final size of the formatted log stream.",
+            "metadata": [{"type": "result"}]
+        },
+        {
+            "name": "cloudwatchlogs_openStreamInEditor",
+            "unit": "Bytes",
+            "description": "Download a log stream to open in editor. Value indicates the final size of the formatted log stream.",
+            "metadata": [{"type": "result"}]
+        },
+        {
+            "name": "cloudwatchlogs_openCurrentInEditor",
+            "unit": "Bytes",
+            "description": "Open currently loaded log stream in editor. Value indicates the number of lines opened.",
+            "metadata": [{"type": "result"}]
+        },
+        {
+            "name": "cloudwatchlogs_wrapLogs",
+            "description": "Word wrap logs off/on",
+            "metadata": [{"type": "enabled"}]
+        },
+        {
+            "name": "cloudwatchlogs_tailLogs",
+            "description": "Tail logs off/on",
+            "metadata": [{"type": "enabled"}]
+        },
+        {
+            "name": "cloudwatchlogs_refreshLogGroup",
```

Comment:
Does this just submit a count of presses or does it also include the total time taken?

Reply:
Just counts the number of presses, I'm skeptical of refresh time because it's more influenced by the service and what is being loaded so I don't think we'll get any data from it.

Reply:
That fair. But I think we should start capturing real performance metrics because it's the best way to see the bottlenecks that might only be visible under certain hardware / network situations.